### DISCUSSION TITLE
Why was my application rejected and how can I reapply susccesfuly?

### DISCUSSION THREAD

**Discussion by elusakin05** [community/community#59550] [GitHub Education]
### Hi everyone,

I understand that my previous application was not approved beacuse my schools name could not be determined,  but I strongly believe that I meet the eligibility criteria and would greatly benefit from accessing the package.

I have been enrolled in MEWAR INTERNATIONAL UNIVERSITY, NIGERIA for a 4 years degree in BSC. Physics With Electronics since September 2022

I have included my official enrollment letter as proof of my  academic standing. However,  my school does not issue student email addresses, and unfortunately, it is not a GitHub Education partner school.
Another problem is that my school was not going on the Github map, but I found it right in the Google map.

I emphasize my genuine need for the GitHub Student Education Package to further develop my skills and knowledge in version control, and software development. The resources and tools provided through the package would greatly enhance my learning experience and enable me to actively contribute to the open-source community.

I kindly request your reconsideration of my application for the GitHub Student Education Package.

**Comment by piincher** [comment]
Hi,Im in the same situation as you hope to get response for this question

**Comment by samus-aran** [comment]
Hi @elusakin05  👋

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.
